app/plugin_api.go

```diff
@@ -6,11 +6,10 @@ package app
 import (
 	"encoding/json"
 	"fmt"
-	"net/http"
-	"strings"
-
 	"github.com/mattermost/mattermost-server/mlog"
 	"github.com/mattermost/mattermost-server/model"
+	"net/http"
+	"strings"
```

Comment:
@chclaus Could you please revert this changes? Importing go's std packages first is pretty common.

Reply:
Hej @hanzei,

this is the result of _gofmt_. I think this should be the standard, shouldn't it?

Best, Christian